src/transformers/tokenization_utils_fast.py

```diff
@@ -112,6 +115,10 @@ def __init__(self, *args, **kwargs):
         elif slow_tokenizer is not None:
             # We need to convert a slow tokenizer to build the backend
             fast_tokenizer = convert_slow_tokenizer(slow_tokenizer)
+        elif from_gguf is not None:
+            # We need to convert a slow tokenizer to build the backend
+            tokenizer_dict = load_gguf_checkpoint(kwargs.get("vocab_file"))["tokenizer"]
```

Comment:
The correct vocab file is always passed here: https://github.com/huggingface/transformers/pull/30391#discussion_r1579298516 so this will less likely to happen, but if one passes `None` indeed it'll fail